src/main/kotlin/controller/LottoController.kt

```diff
@@ -1,51 +1,104 @@
 package controller
```

Comment:
`lotto` 패키지를 기준으로 프로젝트를 관리하면 어떨까요? 아래의 글이 도움 되면 좋겠어요.
https://wikidocs.net/231

```suggestion
package lotto.controller
```